### DISCUSSION TITLE
Fluent UI 9 Combobox Doesn't accept spaces

### DISCUSSION THREAD

**Discussion by Debro012** [microsoft/fluentui#27310] [Q&A] [answered]
I'm utilizing freeform, onChange, and onOptionSelect to control for the Combobox, but I cannot seem to set it to allow freeform that includes spaces. As soon as a space is inserted, the first match is selected. I am seeking the ability to allow users to search for options which include spaces (i.e. names of buildings).

**Comment by behowell** [comment, marked as answer]
Hi @Debro012, this is tracked by https://github.com/microsoft/fluentui/issues/27255. Please follow that issue for more details.
